discovery-infra/tests/test_logs.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import tarfile
+import time
+
+from assisted_service_client.rest import ApiException
+from tests.base_test import BaseTest
+
+class TestDownloadLogs(BaseTest):
```

Comment:
can we add a cleanup method for the downloaded file?

Reply:
We can expect exact number of files only on the happy flow. On the cancel flow, it depends on the host status which differ from run to run. As for cleaning the file, I will add a delete but since they are in /tmp they will be also cleared automatically, eventually